docs/development/upgrading-node.md

```diff
@@ -70,49 +66,35 @@ We need to generate a patch file from each patch applied to V8.
 4. Inspect Node [repo](https://github.com/electron/node) to see what patches upstream Node
   used with their v8 after bumping its version
   - `git log --oneline "deps/v8"`
+  - The commit message "update V8 to <version>" represents replacing `deps/v8` with the named version of V8, exactly as it is in upstream. The commit message "patch V8 to <version>" isn't a complete replacement of `deps/v8`, but just applying the changes in that version.
 5. Create a checklist of the patches. This is useful for tracking your work and for
   having a quick reference of commit hashes to use in the `git diff-tree` step below.
-6. Read `patches/common/v8/README.md` to see which patchfiles came from the previous version of V8 and therefore need to be removed.
-  - Delete each patchfile referenced in `patches/common/v8/README.md`
-7. Apply all patches with the [`get-patch` script](https://github.com/electron/libchromiumcontent/blob/master/script/README.md#get-patch):
+6. Apply all patches with the [`get-patch` script](https://github.com/electron/electron/blob/master/script/README.md#get-patch):
```

Comment:
It shouldn't say "Apply all patches..." but rather "Save all commits as patch files...".

And maybe this whole recommendation should replaced with your solution with `git format-patch --relative=deps/v8 ...`

Reply:
Good idea, I've updated this to use the format-patch command.